upgrade/sql/8.1.0.sql

```diff
@@ -3,3 +3,7 @@ SET NAMES 'utf8mb4';
 
 UPDATE `PREFIX_configuration` SET `value` = 'US/Pacific' WHERE `name` = 'PS_TIMEZONE' AND `value` = 'US/Pacific-New' LIMIT 1;
 DELETE FROM `PREFIX_timezone` WHERE `name` = 'US/Pacific-New';
+
+/* Default configuration for backorder, in order to keep behavior */
+INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES
+  ('PS_ENABLE_BACKORDER_STATUS', '1', NOW(), NOW());
```

Comment:
0 In installation script ?

Reply:
Yes, OFF for new installs, ON for upgraded installs, so there are no surprises for current merchants. 👍

Reply:
OK I agree but there is a conflict

Reply:
rebased ;-)